identity-server/test/IdentityServer.UnitTests/Licensing/v2/DiagnosticEntries/EndpointUsageDiagnosticEntryTests.cs

```diff
@@ -70,7 +70,7 @@ public async Task Should_Handle_Unknown_Endpoints()
         endpointUsage.GetProperty("other").GetInt64().ShouldBe(1);
         foreach (var endpoint in _endpoints)
         {
-            endpointUsage.GetProperty(endpoint).GetInt64().ShouldBe(0);
+            endpointUsage.GetProperty(endpoint.Value!).GetInt64().ShouldBe(0);
```

Comment:
Can we please add a test that verifies that /.well-known/oauth-authorization-server/bogus-unknown-issuer-subpath is counted as an unknown call, and not a disco call?

Reply:
Oh, I guess I was wrong, let's not do anything here then.